src/odin/maneuver.cc

```diff
@@ -622,12 +623,24 @@ void Maneuver::set_unnamed_mountain_bike_trail(bool unnamed_mountain_bike_trail)
   unnamed_mountain_bike_trail_ = unnamed_mountain_bike_trail;
 }
 
-bool Maneuver::verbal_multi_cue() const {
-  return verbal_multi_cue_;
+bool Maneuver::imminent_verbal_multi_cue() const {
```

Comment:
for the Maneuver class - split `verbal_multi_cue` into `imminent_verbal_multi_cue` and `distant_verbal_multi_cue`